schema/operationType.cc

```diff
@@ -33,6 +33,7 @@ namespace minsky
   {
       if (t<add) return general;
       if (t<copy) return binop;
+      if (t<integrate) return constop;      
```

Comment:
This fails, as anything less than integrate will be classified as general on line 34.
Best would be to place the constops after Ravel, and before add, then you can use euler (for general) and add (for constops) here.